sormas-ui/src/main/java/de/symeda/sormas/ui/caze/CaseDataForm.java

```diff
@@ -316,7 +309,7 @@ public class CaseDataForm extends AbstractEditForm<CaseDataDto> {
 	private FollowUpPeriodDto expectedFollowUpPeriodDto;
 	private boolean ignoreDifferentPlaceOfStayJurisdiction = false;
 
-	private final Map<ReinfectionDetailGroup, CaseReinfectionCheckBoxTree> reinfectionTrees = new EnumMap<>(ReinfectionDetailGroup.class);
+	private CheckBoxTreeUpdated<ReinfectionDetail> reinfectionDetailGroupCheckBoxTreeUpdated;
```

Comment:
this field seems to be useless, please convert it to local variable

Reply:
moved the variable declaration in the method